src/events/Duerrewelle.ts

```diff
@@ -14,7 +15,7 @@ export default defineEvent({
   },
 
   probability(game) {
-    return Math.min(1, Math.max(0, (400 - game.values.co2budget) / 400))
+    return linear(100, 0, 400 - game.values.co2budget) / 100
```

Comment:
I think it should be `Math.min(1, linear(400, 0, game.values.co2budget) / 100)`. That will lead to
- Negative probability when the budget is above 400Mt.
- Probability 0 when the budget is at 400Mt.
- Probability 1 when the budget is used up.

The `Math.min(1, ` is needed, because probability above 1 has the special meaning that it happens for certain - which might be ok, but: If the value increases further due to a negative budget, it might increase beyond 3 and prevent a proper game-over event from happening.

...which brings me to enter #119 - which i should have entered last week when stumbling across the same problem.

Reply:
I will merge the PR anyway. Its not going to break anything really important. The event will probably appear quite often in the beginning...

Reply:
...yes, it does. Will fix it directly.